core/src/main/resources/lib/layout/keyboard-shortcut.jelly

```diff
@@ -0,0 +1,56 @@
+<!--
+The MIT License
+
+Copyright (c) 2023, Jenkins Contributors
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define">
+  <st:documentation>
+    <st:attribute name="prefix">
+      Optional prefix to the shortcut
+    </st:attribute>
+    <st:attribute name="suffix">
+      Optional suffix to the shortcut
+    </st:attribute>
+    Displays a keyboard shortcut chip based on the component's body.
+    CMD/CTRL will automatically be translated to the user's operating system's appropriate variant.
+  </st:documentation>
+
+  <div class="jenkins-tooltip__keyboard-shortcut">
+    <j:if test="${attrs.prefix != null}">
```

Comment:
It does not seem to be possible to define empty localizations. An attempt to do so will just use the default locale:

<img width="309" alt="Screenshot 2023-08-29 at 14 29 18" src="https://github.com/jenkinsci/jenkins/assets/1831569/3ce5aea4-83fc-4839-a16e-b66600323a4f">

Reply:
Good spot - updated.